libs/elements/cards/src/lib/components/elewa-news-item-card/elewa-news-item-card.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+import { __highlightedNews as highlightedNewsData } from '@elewa-website/data/sections';
+import { NewsItem } from '@elewa-website/models/schema/ui/cards';
+
+@Component({
+  selector: 'elewa-website-elewa-news-item-card',
+  templateUrl: './elewa-news-item-card.component.html',
+  styleUrls: ['./elewa-news-item-card.component.scss'],
+})
+export class ElewaNewsItemCardComponent implements OnInit  {
+  @Input() newsItem!: NewsItem[]
+  
+  ngOnInit() {
+      this.newsItem = highlightedNewsData
+  }
```

Comment:
Should only render one item. The loop will happen on the parent component to this, which will be created in another issue.